src/PresentationalComponents/NoSystems/NoSystems.js

```diff
@@ -32,26 +28,26 @@ const NoSystems = ({ workloadIs }) => {
         }
     };
 
-    return <Main>
-        <EmptyState variant={ EmptyStateVariant.small } className='insd-c-empty-state-no-systems'>
-            <EmptyStateIcon icon={ workloadTypes[workloadIs]?.icon || ChartSpikeIcon } />
-            <Title headingLevel='h5' size='lg'>
-                {workloadTypes[workloadIs]?.title || intl.formatMessage(messages.noSystemsTitle)}
-            </Title>
+    return <Section>
+        <EmptyState variant={ EmptyStateVariant.sm } className='insd-c-empty-state-no-systems'>
+            <EmptyStateHeader
+                titleText={<>{workloadTypes[workloadIs]?.title || intl.formatMessage(messages.noSystemsTitle)}</>}
+                icon={<EmptyStateIcon icon={ workloadTypes[workloadIs]?.icon || ChartSpikeIcon } />}
+                headingLevel='h5' />
             <EmptyStateBody>
                 <Stack hasGutter>
                     <StackItem>
                         {workloadTypes[workloadIs]?.body || intl.formatMessage(messages.noSystemsDescription)}
                     </StackItem>
                 </Stack>
-            </EmptyStateBody>
-            <InsightsLink app='registration' to="/">
-                <Button variant='primary'>
-                    {intl.formatMessage(messages.registerYourSystems)}
-                </Button>
-            </InsightsLink>
-        </EmptyState>
-    </Main>;
+            </EmptyStateBody><EmptyStateFooter>
```

Comment:
The layout of these JSX seems affected. e.g Should EmptyStateFooter be in the new line?